docs/release-notes.md

```diff
@@ -1,3 +1,23 @@
+## Release v0.5.0
+### Date: April 27, 2023
+This release introduces the Metadata Schema Service and the Policy Service API. Release Automations have been expanded to include Pass-Acceptance-Testing and Pass-Docker.
```

Comment:
```suggestion
This release introduces the Metadata Schema Service and the Policy Service API. Release Automation has been expanded to include [pass-acceptance-testing](https://github.com/eclipse-pass/pass-acceptance-testing) and [pass-docker](https://github.com/eclipse-pass/pass-docker).
```

Reply:
A brief (few word) description of the purpose of the Metadata Schema Service and Policy Service would help make this more understandable for those not already familiar with how PASS works.